.github/workflows/ci_ui.yml

```
@@ -37,6 +37,8 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
+    - name: update chrome
+      run : sudo apt-get update && sudo apt-get install google-chrome-stable
```

```suggestion
      run : sudo apt update && sudo apt install google-chrome-stable
```
use `apt` instead of `apt-get`